mandatory/3-function-output.js

```diff
@@ -3,16 +3,25 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+// This function will return a random number greater than or equal to 1 or less than 10. It will be a decimal. We don't need to pass an argument through this function.
+
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+// This function will combine 2 or more arguments passed through this function.
+// In this function, word1 will be combined with word2 in that order as word2 is inside the brackets.
+// This function will create a new array, it does not change word1 or word2.
```

Comment:
Hey Mandy, I think you've done a great job in solving all the challenges, nice and clean code. The only note I have - concat method returns a string not an array :)

Reply:
Thanks for reviewing my code, Alena! 
I must've taken that bit about arrays from my google search it says: 
"The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array. "

In this exercise we're not working with arrays so you're right here - thanks for pointing that out :)

Reply:
@lazar-eva My mistake, I read the wrong mdn document, I hadn't realised there are two ways to use concat:

1. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat
2. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat

I'll change my answer, thanks again!